### PR TITLE
Upgrade to Splunk 6.3.7

### PR DESCRIPTION
**Pull request by nmattam**
Upgrading Splunk to 6.3.7 to avoid the risk of crashing.

### CODE REVIEW

**Comment by nmattam** [comment]
working on fixing the build failure.

File: Gemfile [hunk 1]
```diff
@@ -5,5 +5,5 @@ gem 'foodcritic', '~> 4.0'
 gem 'rspec', '~> 3.3'
 gem 'chefspec', '~> 4.3'
 gem 'chef-vault'
-gem 'chef', '~> 12.4'
+gem 'chef', '< 12.14'
```
Comment (nmattam) [inline Gemfile]: chef 12.14+ requires ruby 2.2. For now we decided to lock down to <12.14.

**Comment by gravesb** [comment]
+1

**Comment by acharlieh** [comment]
+1

**Pull request merged**